src/lean_spec/subspecs/containers/__init__.py

```diff
@@ -1,19 +1,37 @@
 """The container types for the Lean consensus specification."""
 
-from .block import Block, BlockBody, BlockHeader, SignedBlock
+from .block import (
+    Block,
+    BlockBody,
+    BlockHeader,
+    BlockWithAttestation,
+    SignedBlockWithAttestation,
+)
 from .checkpoint import Checkpoint
 from .config import Config
 from .state import State
-from .vote import SignedVote, Vote
+from .validator import Validator
+from .vote import (
+    AggreagtedAttestations,
+    Attestation,
+    AttestationData,
+    SignedAggreagtedAttestations,
+    SignedAttestation,
+)
 
 __all__ = [
     "Block",
+    "BlockWithAttestation",
     "BlockBody",
     "BlockHeader",
     "Checkpoint",
     "Config",
-    "SignedBlock",
-    "SignedVote",
+    "SignedBlockWithAttestation",
+    "Validator",
+    "AttestationData",
+    "Attestation",
+    "SignedAttestation",
+    "SignedAggreagtedAttestations",
```

Comment:
```suggestion
    "SignedAggregatedAttestations",
```

also applies to other places